{
	"primary_field": "changeset.date",
    "primary_interval": 500000,
	"last_replication_time": "./results/data/repo_last_replication_time.txt",
	"source": {
		"host": "http://klahnakoski-es.corp.tor1.mozilla.com",
		"port": 9200,
		"index": "repo",
		"type": "revision",
		"debug": false
	},
	"destination": {
		"host": "http://activedata.allizom.org",
		"port": 9200,
		"index": "repo",
		"type": "revision",
		"timeout": 300,
		"schema": {
			"$ref": "//.../mo_hg/resources/revision.json"
		},
		"debug": false,
		"limit_replicas": true,
		"zip": false
	},
	"constants": {
		"pyLibrary.env.http.default_headers": {
			"Referer": "https://wiki.mozilla.org/Auto-tools/Projects/ActiveData"
		}
	},
	"debug": {
		"log": [
			{
				"log_type": "stream",
				"stream": "sys.stdout"
			},
			{
				"class" : "logging.handlers.RotatingFileHandler",
				"filename" : "./results/logs/restore_repo.log",
				"maxBytes" : 10000000,
				"backupCount" : 10,
				"encoding" : "utf8"
			},
			{
				"log_type": "ses", "$ref": "file://~/private.json#aws_credentials",
				"from_address": "klahnakoski@mozilla.com",
				"to_address": "klahnakoski@mozilla.com",
				"subject": "[ALERT][Toronto] Problem in restore_repo",


			}
		]
	}
}
